backend/data.js

```diff
@@ -30,11 +30,9 @@ function readFile() {
 
 }
 var response=readFile();
```

Comment:
Why readFile here ? @yamini2697

Reply:
It is not required. Just for testing purpose I have written it.